packages/DAGaml/DAGaml.0.02/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "2.0"
+synopsis: "DAGaml : Abstract DAG manipulation in OCaml"
+maintainer: "Joan Thibault <joan.thibault@ens-rennes.fr>"
+authors: "Joan Thibault <joan.thibault@ens-rennes.fr>"
+license: "LGPL-3.0-only with OCaml-LGPL-linking-exception"
+homepage: "https://gitlab.com/boreal-ldd/dagaml"
+bug-reports: "https://gitlab.com/boreal-ldd/dagaml"
+depends: [
+	"ocaml" {>= "4.08.1" & < "4.12"}
```

Comment:
Just curious, what is the reason 4.12 and up aren’t supported?

Reply:
Testing it locally in 4.13 I’m not seeing any issues
```suggestion
	"ocaml" {>= "4.08.1"}
```

Reply:
Actually, there is a linking-time error with `GuaCaml.0.03` due to the lack of some hash-related function starting from ocaml 4.12.
So, I added a patch to GuaCaml ( `GuaCaml.0.03.01` ) and fixed DAGaml's dependencies accordingly. =)